azurelinuxagent/common/event.py

```diff
@@ -290,7 +290,7 @@ def _add_event(self, duration, evt_type, is_internal, is_success, message, name,
         try:
             self.save_event(json.dumps(data))
         except EventError as e:
-            logger.error("{0}", e)
+            logger.periodic_error(logger.EVERY_FIFTEEN_MINUTES, ustr(e))
```

Comment:
We agreed we would prepend "[PERIODIC]" to the message of periodic events/logs to help during debugging

Reply:
Thanks for the reminder, updated.